Mathlib/MeasureTheory/Function/Holder.lean

```diff
@@ -0,0 +1,160 @@
+import Mathlib.Analysis.InnerProductSpace.Basic
+import Mathlib.Analysis.Normed.Module.Dual
+import Mathlib.Data.Real.StarOrdered
+import Mathlib.MeasureTheory.Integral.Bochner
+import Mathlib.Order.CompletePartialOrder
+
+open ENNReal
+
+class ENNReal.HolderTriple (r : semiOutParam ℝ≥0∞) (p q : ℝ≥0∞) : Prop where
+  one_div_add_eq : 1 / r = 1 / p + 1 / q
```

Comment:
Should we use the simp-normal form here?
```suggestion
  one_div_add_eq : r⁻¹ = p⁻¹ + q⁻¹
```

Reply:
Yes, that seems preferable. I went with `/` originally just because of the statement of [MeasureTheory.Memℒp.mul](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Function/LpSeminorm/CompareExp.html#MeasureTheory.Mem%E2%84%92p.mul), so I'll probably change that while I'm at it.